packages/components/package.json

```diff
@@ -27,7 +27,7 @@
     "build:watch": "npm run build:js -- --watch",
     "build:storybook": "build-storybook -o storybook-static",
     "deploy:docs": "storybook-to-ghpages --ci",
-    "prepack": "npm run build && cp ./package.json dist/",
+    "prepack": "npm run build && cp ./package.json ./tailwind.config.js dist/",
```

Comment:
this includes the `playground/` folder as well ... I didn't think it was a big deal yet but we could rm -rf it here or ignore it when building

Reply:
oh also re: tailwind config, I tested
```
import {theme} from '@chanzuckerberg/components/tailwind.config';
```
in traject, and it was working. could also import the whole config object

Reply:
Regarding the import, I think it would be better to import the theme directly from `@chanzuckerberg/czedi-kit-components`, unless we need to export the whole config for other reasons.

Also, don't have to do it now, but I'd be pretty conservative about what ends up in the published package, and would probably rm the playground folder.

Could do that as part of a build script ran in the build step, isntead of the pack, like we did in https://github.com/chanzuckerberg/frontend-libs/blob/83af6282feefee6cacd3137aa867513a4c6a1f43/scripts/build.sh.

Reply:
yeah good point, I'll rm playground

re: import, in my head I was imagining there's a chance we actually reference the file in some postcss or tailwind compilation step...we might not though, in which case I'd agree with you. I added this context to https://app.clubhouse.io/czi-edu/story/62334/ds-prep-tailwind-config-for-export so we don't forget

Reply:
That's true. We may have to reference that config file somewhere, at some point.